src/Entity/ReferentManagedArea.php

```diff
@@ -80,6 +80,15 @@ public function removeTag(ReferentTag $tag): void
         $this->tags->removeElement($tag);
     }
 
+    public function getOnlyManagedCountryCodes($value): array
+    {
+        return array_values(array_filter(array_map(function (ReferentTag $tag) use ($value) {
+            if (ctype_alpha($tag->getCode()) && 0 === stripos($tag->getName(), $value)) {
+                return [$tag->getCode() => $tag->getName()];
```

Comment:
Why returning a key if at the end we do `array_values` ?

Reply:
It is the subarray keys and not the ones removed by `array_values`.

Reply:
:+1: thank you !